src/client/components/selectable-string-filter-menu/selectable-string-filter-menu.tsx

```diff
@@ -199,7 +199,6 @@ export class SelectableStringFilterMenu extends React.Component<SelectableString
   renderSelectMode(): JSX.Element {
     const { filterMode, onClose, dimension } = this.props;
     const { dataset, selectedValues, searchText } = this.state;
-    const hasMore = isLoaded(dataset) && dataset.dataset.data.length > TOP_N;
```

Comment:
We set `limit` on query that retrieves this dataset. It must go something wrong if we get more than TOP_N records here.